src/gccrs/args.rs

```diff
@@ -202,7 +202,12 @@ impl GccrsArgs {
             .to_owned();
 
         match self.crate_type {
-            CrateType::Bin => args.append(&mut vec![String::from("-o"), output_file]),
+            CrateType::Bin => args.append(&mut vec![
+                String::from("-o"),
+                output_file,
+                String::from("-fPIE"),
```

Comment:
I feel like these should be named constants and maybe even in a already populated array something like:
```
const FPIE
const PIE

default_executable_arguments = [FPIE, PIE]
```

Reply:
We can have a const vector that we add to every set of commands regarding the type of output desired, since I believe that is what we want.